.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+inherit_from: .rubocop_todo.yml
+
+
+AllCops:
+  DisplayCopNames: true
+  Exclude:
+    - 'remediations/**/*'
```

Comment:
We should ask around if anyone uses the remediation stuff in Hydrus at all. I'd guess not, and that all remediation happens with DOR tooling.